ui/pages/notification-details/notification-details-footer/notification-details-footer.tsx

```diff
@@ -29,21 +30,14 @@ export const NotificationDetailsFooter = ({
       padding={4}
       gap={4}
     >
-      {footer.type === 'footer_onchain_notification' && (
-        <>
-          <footer.ScanLink notification={notification} />
-        </>
+      {footer.type === NotificationComponentType.OnChainFooter && (
+        <footer.ScanLink notification={notification} />
+      )}
+      {footer.type === NotificationComponentType.AnnouncementFooter && (
+        <footer.ExtensionLink notification={notification} />
       )}
-      {footer.type === 'footer_feature_announcement' && (
```

Comment:
Is this change intentional?

Reply:
Yes

Reply:
I don't understand then, why are we removing elements from another type of notification?

Reply:
oh shoot that was NOT intentional, good catch.